tests/e2e/workloads/pvc_snapshot_and_clone/test_small_files_pvc_cloned_status.py

```diff
@@ -0,0 +1,153 @@
+import logging
+import random
+import os
```

Comment:
I dont see any steps related to clones, please check and change the test case name accordingly.

Reply:
picked it from bug- 
[Bug 2039265](https://bugzilla.redhat.com/show_bug.cgi?id=2039265) - [Tracker for Ceph [BZ #2039276](https://bugzilla.redhat.com/show_bug.cgi?id=2039276)] CephFS: Failed to create clones if the PVC is filled with smaller size files